package-lock.json

```diff
@@ -3,6 +3,8133 @@
   "version": "0.1.0",
   "lockfileVersion": 1,
   "requires": true,
+  "packages": {
```

Comment:
Please undo the change.

Reply:
how should I do this? am I able to commit to branch `published/juniwang/azure-rest-api-specs-pr/juniwang/wps`?

Reply:
Reverted. Looks like I have enough permission to push to the src branch